lib/features.js

```diff
@@ -52,6 +53,23 @@ module.exports = config => {
     },
 
     /**
+     * Return whether an account meets the account created age requirement to
+     * bypass sign-in confirmation.
+     *
+     * @returns {boolean}
+     */
+    isSigninBypassAccountWithAgeEnabled(account) {
```

Comment:
I flipped flopped several times on whether this should be a `feature`, but decided this was the most logical place to put it.